.travis/test_runner.sh

```diff
@@ -10,13 +10,14 @@
 
 set -e
 
-TESTS="$1"
-COMMAND="coverage run manage.py test --noinput --failfast --traceback --verbosity=2"
+# --divide-depth=1 to descend into django-nose database contexts
+# --divide-depth is ignored if --divided-we-run is not specified
+COMMAND="coverage run manage.py test --noinput --stop --verbosity=2 --divide-depth=1"
 
 /moto-s3/env/bin/moto_server s3 &
 
 if [ -z ${COMMAND_OVERRIDE} ]; then
-    $COMMAND $TESTS
+    $COMMAND "$@"
```

Comment:
This passes all test arguments instead of only the first one. Most of the shard tests have not been running and are pretty broken.